lib/main_common.pm

```diff
@@ -157,8 +157,14 @@ sub load_autoyast_tests {
     else {
         loadtest("autoyast/repos");
         loadtest("autoyast/clone");
+        if (get_var('SP3ORLATER') && check_var('FILESYSTEM', 'btrfs')) {
+            loadtest "autoyast/verify_autoinst_btrfs";
+        }
         loadtest("autoyast/logs");
     }
+    if (get_var('SP3ORLATER') && check_var('FILESYSTEM', 'btrfs')) {
```

Comment:
I found it in lib/registration.pm and in main.pm we have 
if (sle_version_at_least('12-SP3')) {
    set_var('SP3ORLATER', 1);
}

Should I change it in both places then?